content/en/training/_index.html

```diff
@@ -112,7 +112,6 @@ <h2>Kubernetes Training Partners</h2>
     </center>
   </div>
   <div class="main-section landscape-section">
-    <iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no"></iframe>
-    <script src="https://landscape.cncf.io/iframeResizer.js"></script>
+    {{< cncf-landscape helpers=false category >}}
```

Comment:
```suggestion
    {{< cncf-landscape helpers="false" category="kubernetes-training-partner" >}}
```

ought to do it

Reply:
Thanks!
@sftim @kbhawkey 
revirew page 
https://deploy-preview-24014--kubernetes-io-master-staging.netlify.app/training/